docs/src/build_config/passthrough/hypervisor_options.md

```diff
@@ -0,0 +1,110 @@
+# Microvm hypervisor options
+
+Microvm is the component responsible for launching virtual machines.
```

Comment:
Technically `microvm.nix` is only defining the launch services generated for `systemd.`

`microvm.nix` is "A Nix Flake to build NixOS and run it on one of several Type-2 Hypervisors on NixOS/Linux." [1] and `systemd` is responsible for launching the `microvm.nix`-generated (built) systemd services for guest VMs (based on NixOS) defined with `microvm.nix` [2]

[1] https://github.com/astro/microvm.nix#:~:text=A%20Nix%20Flake%20to%20build%20NixOS%20and%20run%20it%20on%20one%20of%20several%20Type%2D2%20Hypervisors%20on%20NixOS/Linux.
[2] https://astro.github.io/microvm.nix/host-systemd.html

Reply:
OK